components/badges/BadgeBox.tsx

```diff
@@ -0,0 +1,33 @@
+import { Link, Box, Text, Badge, Image } from '@chakra-ui/react'
+import { ethers } from 'ethers'
+import useTranslation from 'next-translate/useTranslation'
+import { displayValue } from '@/utils/bigNumber'
+import { useFetchTokenURIJSON } from '@/hooks/badge/useFetchMetaData'
+import { useEffect, useState } from 'react'
+
+export const BadgeBox: React.FC<{
+  badge: [boolean, boolean, ethers.BigNumber, ethers.BigNumber, string]
+  tokenId: number
+}> = (data) => {
+  const [, , amount, maxSupply, tokenURI] = data.badge
+  const { t } = useTranslation('common')
+  const { tokenURIJSON } = useFetchTokenURIJSON(tokenURI)
+
+  if (!tokenURIJSON) {
+    return <></>
+  }
+
+  return (
+    <Link href={`/badges/${data.tokenId}`}>
+      <Box boxShadow="xs" p="6" rounded="md" bg="whiteAlpha.100">
+        <Image mt={1} src={tokenURIJSON.image} alt={t('IMAGE_PREVIEW_ALT')} />
+        <Badge variant="outline" colorScheme="green">
+          Max Supply: {maxSupply.toString()}
+        </Badge>
```

Comment:
if it would be easy `number of mint / maxSuply ` would be the best.
it donest need to be now 
https://github.com/henkaku-center/badge/blob/main/contracts/HenkakuBadge.sol#L25

Reply:
fixed https://github.com/henkaku-center/omise-interface/pull/102/commits/d7b29d81b41a32d81a510d1b6c6035846fd24a91